### DISCUSSION TITLE
Get window.location.hostname within the userChrome scripts?

### DISCUSSION THREAD

**Discussion by clifford269** [Aris-t2/CustomJSforFx#111] [General]
Currently it returns chrome://browser...

**Comment by clifford269** [comment]
Found out myself

**Comment by Aris-t2** [comment]
So, what was the solution?
(Did not have the time yet to look into it.)

**Reply by clifford269** [reply]
Using gBrowser. You've used it yourself.

**Reply by Aris-t2** [reply]
Thanks, I thought something might have changed.

**Reply by clifford269** [reply]
I'm taking this up in order to bring the Frutiger Aero goodness to FF. It will be launched as a repository soon.
Any chance you're interested in Linux development, Aris?

**Reply by Aris-t2** [reply]
I do not have enough free time to look into more projects. ;-)
